tajo-core/src/main/java/org/apache/tajo/session/Session.java

```diff
@@ -121,6 +124,10 @@ public synchronized String getCurrentDatabase() {
     return currentDatabase;
   }
 
+  public synchronized void setQueryCache(LoadingCache<String, Expr> cache) { this.cache = cache; }
```

Comment:
That's not true. According to @hyunsik 's advice,it's a sesstion based cache. It is set once for each sesstion.